src/components/playback/playbackmanager.js

```diff
@@ -2601,8 +2607,7 @@ class PlaybackManager {
                 playOptions.items = null;
 
                 return getPlaybackMediaSource(player, apiClient, deviceProfile, item, mediaSourceId, options).then(async (mediaSource) => {
-                    const user = await apiClient.getCurrentUser();
-                    autoSetNextTracks(prevSource, mediaSource, user.Configuration.RememberAudioSelections, user.Configuration.RememberSubtitleSelections);
+                    autoSetNextTracks(prevSource, mediaSource, rememberAudio, rememberSubtitle);
```

Comment:
Calling `autoSetNextTracks` after `getPlaybackInfo` still breaks selected subtitle track when transcoding.

I tested with this set:
```
'Bleach S01E01.en.alt.srt'
'Bleach S01E01.en.srt'
'Bleach S01E01.mp4'
'Bleach S01E01.ru.alt.srt'
'Bleach S01E01.ru.srt'
'Bleach S01E01.srt1.srt'
'Bleach S01E01.srt2.srt'
'Bleach S01E02.en.srt'
'Bleach S01E02.mp4'
'Bleach S01E02.ru.srt'
'Bleach S01E02.srt1.srt'
'Bleach S01E02.srt2.srt'
```
1. Enable burn-in for all subtitles.
2. Start `S01E01`.
3. Select `Ru Alt`.
4. Skip to the next item.
5. It encodes `str1`, but `Ru` is selected.

As I said, I think we should only call `autoSetNextTracks` before `getPlaybackInfo`. 
This way we don't alter the audio and subtitle tracks that are used for the remuxing/transcoding.
But to do that, we need to get media streams first.